extensions/search-composer-packagist/src/composer.tsx

```diff
@@ -0,0 +1,111 @@
+import { ActionPanel, CopyToClipboardAction, List, OpenInBrowserAction, showToast, ToastStyle } from "@raycast/api";
+import { useEffect, useMemo, useState } from "react";
+import algoliaSearch from "algoliasearch/lite";
+
+const APPID = "M58222SH95";
+const APIKEY = "5ae4d03c98685bd7364c2e0fd819af05";
+const INDEX = "packagist";
+
+type PackagistHitMeta = {
+  downloads: number;
+  downloads_formatted: string;
+  favers: number;
+  favers_formatted: string;
+};
+
+type PackagistHit = {
+  id: number;
+  name: string;
+  package_organisation: string;
+  package_name: string;
+  description: string;
+  type: string;
+  repository: string;
+  language: string;
+  trendiness: number;
+  popularity: number;
+  meta: PackagistHitMeta;
+  tags: string[];
+};
+
+function getComposerRequireCommand(hit: PackagistHit) {
+  return `composer require ${hit.name}`;
+}
+
+function getComposerRequireDevCommand(hit: PackagistHit) {
+  return `composer require --dev ${hit.name}`;
+}
+
+function getPackagistPageURL(hit: PackagistHit) {
+  return `https://packagist.org/packages/${hit.name}`;
+}
+
+export default function SearchDocumentation() {
+  const algoliaClient = useMemo(() => {
+    return algoliaSearch(APPID, APIKEY);
+  }, [APPID, APIKEY]);
+
+  const algoliaIndex = useMemo(() => {
+    return algoliaClient.initIndex(INDEX);
+  }, [algoliaClient, INDEX]);
+
+  const [searchResults, setSearchResults] = useState<PackagistHit[] | undefined>();
+  const [isLoading, setIsLoading] = useState(false);
+
+  const search = async (query = "") => {
+    if (query === "") {
+      return [];
+    }
+    setIsLoading(true);
+    try {
+      const res = await algoliaIndex.search<PackagistHit>(query, {
+        hitsPerPage: 30,
+        facets: ["tags", "type", "type"],
+      });
+      setIsLoading(false);
+      return res.hits;
+    } catch (err: any) {
+      setIsLoading(false);
+      await showToast(ToastStyle.Failure, "Error Searching Composer Packagist.", err.message);
+      return [];
+    }
+  };
+
+  useEffect(() => {
+    (async () => setSearchResults(await search()))();
+  }, []);
+
+  return (
+    <List
+      throttle={true}
+      isLoading={isLoading}
+      onSearchTextChange={async (query) => setSearchResults(await search(query))}
+    >
+      {searchResults?.map((hit: PackagistHit) => {
+        return (
+          <List.Item
+            key={hit.id}
+            title={hit.name}
+            subtitle={hit.description}
```

Comment:
```suggestion
            subtitle={hit.description}
            accessoryIcon={Icon.Star}
            accessoryTitle={hit.meta.favers_formatted}
```
You could add more awesomeness! 💪

Reply:
Thanks @pernielsentikaer , I was searching for it 💪🏻 😄